rebar.config

```diff
@@ -1,7 +1,7 @@
 {deps, [ {kafka_protocol, "4.1.5"}
        , {snappyer, "1.2.9"}
        ]}.
-{project_plugins, [{coveralls, "~> 2.2.0"}, {rebar3_lint, "~> 1.0.2"}]}.
+{project_plugins, [{coveralls, "~> 2.2.0"}, {rebar3_lint, "~> 3.2.5"}]}.
```

Comment:
`rebar3_lint` depends on `katana_code` that would fail when imported in `1.0.2`.